apps/api/infra/api.ts

```diff
@@ -190,6 +190,11 @@ export const serviceSetup = (services: {
         staging: 'https://api-staging.thinglysing.is/business/tolfraedi',
         prod: 'https://api.thinglysing.is/business/tolfraedi',
       },
+      FORM_SYSTEM_API_BASE_PATH: {
+        dev: 'https://profun.island.is/umsoknarkerfi',
+        staging: 'https://profun.island.is/umsoknarkerfi',
+        prod: 'https://profun.island.is/umsoknarkerfi',
```

Comment:
Feel like this is a bit dangerous and a typical thing that forgets to be updated before moving to those environments. So we should maybe leave staging and prod as empty strings until or a string placeholder like `This needs to be updated before using in this environment`?